.github/workflows/test.yml

```diff
@@ -0,0 +1,78 @@
+on:
+  workflow_call:
+    inputs:
+      name:
+        description: Name of the driver test
+        required: true
+        type: string
+      driverRepoUrl:
+        description: Git URL of the driver repository
+        required: true
+        type: string
+      driverRepoBranch:
+        description: Driver repository branch to check out
+        required: true
+        type: string
+      php:
+        description: PHP version for running tests
+        required: true
+        type: string
+      setUpCmd:
+        description: Command to run before test
+        required: false
+        type: string
+      testCmd:
+        description: Command that runs the test
+        required: true
+        type: string
+      tearDownCmd:
+        description: Command to run after test
+        required: false
+        type: string
```

Comment:
Maybe some of the form fields (e.g. `php` and `name`) should really be of `choice` type instead of `string` type.

Reply:
Nah.

The `name` field is open text (each test name could be anything, in one case we even have 2 tests (with a small variation) for the same driver).
There are too many `php` versions to list down. I thought about using a numeric type, but that's just weird (for a version). :)

But good point, we could do that if/when something more specific comes up.